example/main.go

```diff
@@ -2,14 +2,31 @@ package main
 
 import (
 	"fmt"
+	"os"
+	"path/filepath"
+	"runtime"
 
-	"github.com/getlantern/pacon"
+	"github.com/getlantern/pac"
 )
 
 func main() {
-	pacon.PacOn("localhost:12345/pac")
-	fmt.Println("proxy set, any key to continue...")
+	pac.SetHelperPath(absPath("./pac-cmd"))
+	if runtime.GOOS == "darwin" {
+		pac.SetIconPathOnOSX(absPath("icon.png"))
+		pac.SetPromptOnOSX("Input your password and save the world!")
+	}
+	err := pac.On("localhost:12345/pac")
+	if err != nil {
+		fmt.Printf("Error set proxy: %s\n", err)
+		return
+	}
+	fmt.Println("proxy set, Enter continue...")
 	var i int
 	fmt.Scanf("%d\n", &i)
-	pacon.PacOff()
+	pac.Off()
+}
+
+func absPath(name string) string {
+	wd, _ := filepath.Abs(filepath.Dir(os.Args[0]))
```

Comment:
What if I don't want to put the pac file in the same location as my binary?  In particular, on OS X we should put it in ~/Library/Lantern.  On Windows it should go in AppData\Lantern.